addons/web/static/src/legacy/legacy_views.js

```diff
@@ -18,6 +19,188 @@ function getJsClassWidget(fieldsInfo) {
     return legacyViewRegistry.get(key);
 }
 
+/**
+ * @param {string} state
+ * @param {string} resModel
+ * @returns {string}
+ */
+function legacyToNew(state, resModel) {
```

Comment:
function should be renamed I think. It doesn't really say what it does.

Reply:
searchStateFromLegacy